docs/md/updates/update-1.md

```diff
@@ -9,32 +9,28 @@
 
 # Update #1 -  June 2020
 
-About a month ago we first announced `coding.blog`. It was received really
-well by the community, much more enthusiastically than what we anticipated.
-Since then, we've been at work to more sharply outline our roadmap and work
-towards it.
+About a month ago, we first announced `coding.blog`. It was received really
+well by the community, much more enthusiastically than what we had anticipated.
+So since then, we've been at work to more sharply outline our roadmap and work
+towards making coding.blog a reality. 
 
-Here is a rundown of what has happened since and what are our future plans:
+Here is a rundown of what has happened since our announcement and our future plans:
 
 - The [first blog](https://eugene.coding.blog) was released alongside a
 [plugin for **CODEDOC**](https://connect-platform.github.io/coding-blog-plugin) 
-that helps with creating more elegant blogs, both on `coding.blog` and on other hosting
+that helps developers create more elegant blogs about coding both on `coding.blog` and on other hosting
 services (or for self-hosted blogs).
 
-- We've started the **alpha stage**: we are slowly activating blogs of prospective
-creators, first batch already activated and began testing and publishing their blogs.
+- We've started the **alpha stage**: we are slowly activating blogs of writers who signed up on our prospective
+creators list (you can still sign-up: https://coding.blog/creators). The first batch of creators already have their blogging space set up and have begun testing and publishing their blogs.
```

Comment:
@Bwaris94 this is not how you write links in markdown. simply use this syntax:

`[The display text](the-link)`

for example, `you can still sign-up [here](https://coding.blog/creators)`.

Besides, since this page is on the same website, you can refer to another page via relative URLs, i.e.

`you can still sign-up [here](/creators)`